mantidproject/mantid
# Conda: CI Changes to support nightly master builds

**Description of work:**
The work is mostly in the CI layer to support nightly CI in both RHEL and Ubuntu, using ISIS machines due to it installing things on the agents if it's not already present.

AFTER THIS IS MERGED THE TARGET BRANCH MUST CHANGE TO MASTER IN THE CONFIG FOR THE TWO NEW JENKINS JOBS.

**To test:**
Observe the two projects on Jenkins:
https://builds.mantidproject.org/view/Conda%20Env/job/master_clean_conda_env_linux_ubuntu/
https://builds.mantidproject.org/view/Conda%20Env/job/master_clean_conda_env_linux_rhel7/

Fixes #31617 

Issues raised from this PR: #32088

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
